girder_worker/__main__.py

```diff
@@ -6,11 +7,14 @@
 
 
 def main():
-    try:
-        include_core_tasks = config.getboolean(
-            'girder_worker', 'core_tasks')
-    except (NoSectionError, NoOptionError):
-        include_core_tasks = True
+    if six.PY2:
+        try:
+            include_core_tasks = config.getboolean(
+                'girder_worker', 'core_tasks')
+        except (NoSectionError, NoOptionError):
+            include_core_tasks = True
+    else:
+        include_core_tasks = False
```

Comment:
Would it make sense to print a warning if core tasks are enabled in the config in PY3?

Reply:
hopefully all of the logic related to ```core_tasks``` is going away in the next six to eight weeks.  For the most part i doubt people will be running girder_worker in python3 anyways.  This is mostly to prevent core tasks from being imported in girder's python3 test suite